healthScore/healthScore/templates/view_history.html

```diff
@@ -88,22 +88,23 @@ <h2 class="text-center">Your Health History</h2>
     <div class="row">
         <div class="col-lg-12">
             <h4>Filter Your Health History</h4>
-            <form method="GET" action="{% url 'view_health_history' %}" class="form-inline">
+            <form method="POST" action="{% url 'view_health_history' %}" class="form-inline">
```

Comment:
Could we have it as a GET method itself since we are only retrieving information here? Post is usually used when we need to create new entities